package/wpe/wpeframework-netflix/wpeframework-netflix.mk

```diff
@@ -25,25 +25,25 @@ WPEFRAMEWORK_NETFLIX_CONF_OPTS += -DBUILD_REFERENCE=${WPEFRAMEWORK_NETFLIX_VERSI
 # TODO: Do not have WPEFRAMEWORK versioning yet
 # WPEFRAMEWORK_NETFLIX_CONF_OPTS += -DPLUGIN_NETFLIX_VERSION="$(WEBBRIDGE_BUILD_VERSION)-dev"
 
-ifeq ($(BR2_PACKAGE_PLUGIN_NETFLIX_NETFLIX_AUTOSTART),y)
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART),y)
```

Comment:
Are you sure you updated all defconfigs using these Variables?

Reply:
@MFransen69  I have found out that 1 more BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_PLUGIN flag was set, and is corrected in the next config.
Regarding defconfigs. Currently all defconfigs are using 78:BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_* flag which actually did not applied ;)

Here is a quick search results:
```
dev@ubuntu:~/repos/buildroot $ ag BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX
configs/dawn7002_wpe_defconfig
83:BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART=y

configs/spectrum_wpe_defconfig
86:BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART=y

configs/raspberrypi3-pitop_wpe_ml_defconfig
86:BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_FULLHD=y

configs/bcm7428_wpe_ml_defconfig
78:BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART=y

package/wpe/wpeframework-netflix/wpeframework-netflix.mk
28:ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART),y)
34:ifneq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_MODEL),)
35:WPEFRAMEWORK_NETFLIX_CONF_OPTS += -DPLUGIN_NETFLIX_MODEL="$(call qstrip,$(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_MODEL))"
38:ifneq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_SUSPEND_TIMEOUT),)
39:WPEFRAMEWORK_NETFLIX_CONF_OPTS += -DPLUGIN_NETFLIX_SUSPENDTIMEOUT="$(call qstrip,$(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_SUSPEND_TIMEOUT))"
42:ifneq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_RESUME_TIMEOUT),)
43:WPEFRAMEWORK_NETFLIX_CONF_OPTS += -DPLUGIN_NETFLIX_RESUMETIMEOUT="$(call qstrip,$(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_RESUME_TIMEOUT))"
46:ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_FULLHD),y)

package/wpe/wpeframework-netflix/Config.in
9:config BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_AUTOSTART
14:config BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_MODEL
19:config BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_SUSPEND_TIMEOUT
29:config BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_RESUME_TIMEOUT
39:config BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_NETFLIX_FULLHD
```

And to be 100% sure `dev@ubuntu:~/repos/buildroot $ ag BR2_PACKAGE_WPEFRAMEWORK_PLUGIN_PLUGIN` gives no findings